write-api-client/src/main/java/io.dockstore.client/cli/Add.java

```diff
@@ -0,0 +1,176 @@
+package io.dockstore.client.cli;
```

Comment:
This comment is specifically for the file paths. 
`write-api-client/src/main/java/io.dockstore.client/cli/Add.java ` is odd and should probably be `write-api-client/src/main/java/io/dockstore/client/cli/Add.java` by convention